resources/web/wwi/images/icons/webotsview.svg

```diff
@@ -0,0 +1,9 @@
+<svg version="1.1" id="webotsview" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
To be consistent with other svg names, e.g., robot-window, world-selection, etc., I would rename this one to webots-view.svg